src/DocumentFormat.OpenXml/Packaging/OpenXmlPackage.cs

```diff
@@ -237,7 +237,7 @@ private void Load()
 
         internal virtual ApplicationType ApplicationType => ApplicationType.None;
 
-        internal bool StrictTranslation { get; set; } = false;
+        public bool StrictRelationshipFound { get; set; } = false;
```

Comment:
This needs XML docs, and the `set` needs to be `internal`

Reply:
Done.